x-pack/filebeat/module/crowdstrike/falcon/test/falcon-events.log-expected.json

```diff
@@ -1,113 +1,125 @@
 [
     {
         "@timestamp": "2020-02-19T08:30:00.000Z",
-        "process.pid": 38684386611,
```

Comment:
This logs was already out of sync, that's why there is such a big diff. I ran the update from a clean master and it produced a diff.